kodilla-patterns/src/main/java/com/kodilla/patterns/prototype/library/Book.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.patterns.prototype.library;
```

Comment:
this module is not valid gradle module, I cannot build it, please add missing files.